server/bin/pbench-dispatch.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+
+""" 
+# check directory where tarballs are there
```

Comment:
"... where tarballs are received."

Reply:
But I don't think that's right: the shim does that and moves tarballs to the archive. This script figures out what it has to do by scanning the TODO directories in the archive area.